src/main/java/org/minions/devfund/chelo/WhackAMole.java

```diff
@@ -0,0 +1,242 @@
+            score++;
+            molesLeft--;
+        }
+        attemptsLeft--;
+    }
+
+    /***
+     * A method to print the grid without showing where the moles are.
+     * For every spot that has recorded a whacked mole, print out a W, or * otherwise.
+     * @return string builder
+     */
+    String printGridToUser() {
+        StringBuilder builder = new StringBuilder();
+
+        for (int i = 0; i < moleGrid.length; i++) {
+            for (int j = 0; j < moleGrid.length; j++) {
+                if (moleGrid[i][j] == 'M') {
+                    builder.append("[*]");
+                } else {
+                    builder.append("[" + moleGrid[i][j] + "]");
+                }
+            }
+            builder.append("\n");
+        }
+        return builder.toString();
+    }
+
+    /**
+     * A method to print the grid without showing where the moles are.
+     * For every spot that has recorded a whacked mole print out a W, or * otherwise.
+     * @return string builder
+     */
+    String printGrid() {
+        StringBuilder builder = new StringBuilder();
+
+        for (int i = 0; i < moleGrid.length; i++) {
+            for (int j = 0; j < moleGrid.length; j++) {
+                builder.append("[" + moleGrid[i][j] + "]");
+            }
+            builder.append("\n");
+        }
+        return builder.toString();
+    }
+
+    /**\
+     * A method to return Attempts Left value.
+     * @return remaining attempts
+     */
+    public int getAttemptsLeft() {
+        return attemptsLeft;
+    }
+
+    /**\
+     * A method to return mole Grid Length value.
+     * @return moleGrid
+     */
+    public int getMoleGridLength() {
+        return moleGrid.length;
+    }
+
+    /**
+     * A method to return mole Grid value given x and y.
+     * @param x row
+     * @param y column
+     * @return value
+     */
+    public int getMoleGridValue(int x, int y) {
+        return moleGrid[x][y];
+    }
+
+    /**\
+     * A method to return score value.
+     * @return score
+     */
+    public int getScore() {
+        return score;
+    }
+
+    /**
+     * A method to return moles left value.
+     * @return remaining moles
+     */
+    public int getMolesLeft() {
+        return  molesLeft;
+    }
+
+    /**
+     * A method to print the current status for.
+     * Score,
+     * Attempts Left,
+     * Moles Left.
+     */
+    public void printStatus() {
+        System.out.println("Score: " + this.score
+                + ", Attempts left: " + this.attemptsLeft
+                + ", Moles left: " + this.molesLeft);
+    }
+
+    /**
+     * Main method to run the program.
+     * @param args main method
+     */
+    public static void main(final String[] args) {
+        Scanner scanner = new Scanner(System.in, "UTF-8");
+        System.out.println("Enter number of attempts:");
+        int numAttempts = scanner.nextInt();
+        System.out.println("Enter dimension for the grid:");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)